docs/docs/reference/changed-features/compiler-plugins.md

```diff
@@ -1,7 +1,9 @@
 ---
-layout: doc-page
 title: "Changes in Compiler Plugins"
-movedTo: https://docs.scala-lang.org/scala3/reference/changed-features/compiler-plugins.html
```

Comment:
don't we need to keep the `movedTo`?

Reply:
No, the idea is that we will stop updating the Dotty website.

Maybe just before doing that we should remove the link “Edit this page on GitHub” from the layout, as suggested by @unkarjedy

Reply:
I opened this https://github.com/lampepfl/dotty/issues/13355